packages/lib/services/interop/InteropService.ts

```diff
@@ -53,7 +53,7 @@ export default class InteropService {
 				{
 					...defaultImportExportModule(ModuleType.Importer),
 					format: 'md',
-					fileExtensions: ['md', 'markdown', 'txt'],
+					fileExtensions: ['md', 'markdown', 'txt', 'html'],
```

Comment:
This is to support html type notes.

Reply:
For Markdown files, we generate a file like this if I'm not mistaken?

```markdown
# Note title

Note body......
```

but then for HTML, what do we generate? Because the below wouldn't be valid HTML?

```html
Note title

<html><head>.............
```

Reply:
That's a good point, as is it won't be generating valid html. 
Currently for both it just uses the serialize for edit function, which outputs the below.
```markdown
Note title

Note body......
```

I'm wondering if the best course of action is to just remove the title from the body at all. The current MD importer takes the title from the filename anyways. And users will be using this to import from other apps that don't necessarily export the title in the first line. I think in this case it's worth the data loss (special characters are stripped). The alternative will be to include the title information in the html `<meta>`, which I plan to do anyways for the yaml export/import. I could also update the serializeForEdit function to include a leading # which I think makes sense.

Let me know what you think. I'm leaning towards just removing the title from either body.

Reply:
For now, I've removed the exported title from the markdown body. I'm still wondering what the best move is, but I'll keep thinking about it.